TestingTool/task.py

```diff
@@ -7,4 +7,20 @@
 
 
 class Task(EmbeddedDocument):
-    pass  # I'm just a placeholder
+    name = StringField(max_length=128, required=True)
+    action = StringField(max_length=128, required=True)
+    args = DictField()
+    reverse_result = BooleanField(default=False)
+    result = EmbeddedDocumentField(Result)
+
+
+def run(self, browser_instance):
+    data = DictField()
+    result = BooleanField()
```

Comment:
Nu ar trebui folosite field-uri in afara declaratiilor din clasa (cum ai facut mai sus).
Aici ar trebui folosite obiecte native python, cum ar fi dict sau bool